great_expectations/render/renderer/inline_renderer.py

```diff
@@ -18,13 +19,20 @@
 class InlineRenderer(Renderer):
     def __init__(
         self,
-        render_object: ExpectationValidationResult,
+        render_object: Union[ExpectationConfiguration, ExpectationValidationResult],
     ) -> None:
         super().__init__()
 
-        self._render_object = render_object
+        if isinstance(render_object, ExpectationConfiguration) or isinstance(
+            render_object, ExpectationValidationResult
+        ):
+            self._render_object = render_object
+        else:
+            raise InvalidRenderedContentError(
+                f"InlineRenderer can only be used with an ExpectationConfiguration or ExpectationValidationResult, but {type(render_object)} was used."
+            )
 
-    def get_atomic_rendered_content_for_object(
```

Comment:
Is this a breaking change we should be concerned about?

Reply:
There is no reason for any client to access this, which is why I took it private before we start actually leveraging the renderer.